wakeonlan.py

```diff
@@ -27,8 +27,14 @@ def create_magic_packet(macaddress: str) -> bytes:
     if len(macaddress) == 17:
         sep = macaddress[2]
         macaddress = macaddress.replace(sep, "")
-    elif len(macaddress) != 12:
+    if len(macaddress) == 14:
```

Comment:
Processing an invalid string of length 17 may yield a string of length 14. To avoid further processing this, `elif` should be used.

```suggestion
    elif len(macaddress) == 14:
```